SECURITY.md

```diff
@@ -0,0 +1,12 @@
+# Security Policy
+
+## Supported Versions
```

Comment:
Also see the google OSS policy https://opensource.google/documentation/policies/library-breaking-change. There is a major version backport policy

Reply:
updated